lib/terminus.spec.js

```diff
@@ -136,16 +136,79 @@ describe('Terminus', () => {
     })
 
     it('returns 503 once signal received', (done) => {
-      execFile('node', ['lib/standalone-tests/terminus.onsignal.fail.js'])
+      let responseAssertionsComplete = false
+
+      // We're only truly finished when the response has been analyzed and the forked http process has exited,
+      // freeing up port 8000 for future tests
+      execFile('node', ['lib/standalone-tests/terminus.onsignal.fail.js'], (error) => {
+        expect(responseAssertionsComplete).to.eql(true)
+        done()
```

Comment:
shouldn't we simply pass in the error to the `done(error)`?

Reply:
Ah yes, my original code was kind of unintuitive...

Since we are issuing SIGTERMs, the exit code is actually an error code, but that's what we expect so we don't want to pass it along to `done()`. My code was unintuitively appeasing the lint rule [handle-callback-err](https://eslint.org/docs/rules/handle-callback-err ) via:
```
        if (error) {
          throw error
        }
```

I updated the code to be more obvious how it is snaking around that lint rule.

Reply:
Ooh actually I thought of a better way.. I'll perform a basic assertion against the error, to make it even more explicit that it is expected.